release_notes/v2.1.0.md

```diff
@@ -0,0 +1,14 @@
+# Money this way please!
+
+## Buy actions
+
+![money](https://media0.giphy.com/media/8LDwLDu12y6Fa/200_s.gif)
+
+You can now request money from your users. Payments are handled through [Stripe](https://stripe.com/), and setup is extremely simple. Read all the details [in our docs](http://docs.smooch.io/javascript/#stripe).
+
+
+## Changes
+- The widget was fully rewritten using React and Redux. The public API is the same, but the events published slightly changed. The widget was previously publishing `message:received` and `message:sent`. Now, it only publishes `message` whenever a new message is added to the conversation. You can always check who sent that message by checking the `role` attribute for the message.
```

Comment:
Don't forgot to update this once you add back the `message:received` and `message:sent` events